doc/whats_new/upcoming_changes/sklearn.utils/30516.fix.rst

```diff
@@ -0,0 +1,4 @@
+- Raise a `FutureWarning` when there is no concrete implementation of `__sklearn_tags__`
```

Comment:
why do we not want to show this to users? As in, `FutureWarning` instead of `DeprecationWarning`